sunpy/map/maputils.py

```diff
@@ -81,6 +81,14 @@ def map_edges(smap):
 
 
 @u.quantity_input
+def _is_coordinates_frame_helioprojective(coordinates):
```

Comment:
I would also add a documentation string.

Reply:
So shall I leave it blank?

Reply:
Blank?

Reply:
I meant that if you will be adding the documentation string so I should leave it for now.

Reply:
You should add a documentation string.

Reply:
Sure, I will add a documentation string.